packages/plugin-essentials/sources/commands/set/version.ts

```diff
@@ -86,6 +89,8 @@ export default class SetVersionCommand extends BaseCommand {
     let bundleUrl: string;
     if (this.version === `self`)
       bundleUrl = getBundlePath();
+    else if (this.version.match(/^(http[s]?:)/))
```

Comment:
It would be better to use the same regexp as in [`yarn plugin import`](https://github.com/yarnpkg/berry/blob/4033a1b8f70e0de3b5bee0ba3fe595aed7c25e88/packages/plugin-essentials/sources/commands/plugin/import.ts#L69) for consistency.